cinema/serializers.py

```diff
@@ -1,6 +1,9 @@
+from django.db import transaction
 from rest_framework import serializers
 
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from cinema.models import (
+    Genre, Actor, CinemaHall, Movie, MovieSession, Order, Ticket
+)
```

Comment:
please fix imports as recommended in checklist 
![image](https://github.com/mate-academy/py-tickets-orders/assets/83316497/8981ea6d-661c-46fe-a3a8-6ea6e1031d7a)